docs/usage/security-modes.md

```diff
@@ -0,0 +1,52 @@
+# Security modes
+
+This section describes the different **Security mode** options for connectivity that can be selected for Liqo, regulating connectivity restriction for intra-cluster traffic.
+
+## Overview
+
+Actually, Liqo provides two security modes:
+
+* **Full pod-to-pod connectivity**: represents the native behaviour of Liqo, there are no restriction in terms of connectivity, and all pods of the first cluster can connect to all pods of the second cluster (and vice versa).
+* **Intra-cluster traffic segregation**: full pod-to-pod connectivity is enabled only within real clusters; regarding intra-cluster traffic, local cluster has the possibility to contact only its offloaded pods, while remote cluster is allowed to contact exclusively services offloaded on it
```

Comment:
* **Intra-cluster traffic segregation**: the full pod-to-pod connectivity is allowed only within physical cluster, while in the intra-cluster traffic, the local cluster can contact only its offloaded pods, while the remote cluster is allowed to contact exclusively the services offloaded on it. This mode can be used when the local cluster aims to limit the remote cluster's connectivity solely to offloaded services, preventing direct interaction with the offloaded pods.